kolibri/plugins/learn/assets/src/views/ProgressBar.vue

```diff
@@ -9,18 +9,35 @@
     :aria-valuenow="progress * 100"
   >
     <p
-      v-if="completed"
       class="completion-label"
       :style="{ color: $themePalette.grey.v_800 }"
     >
-      <ProgressIcon
-        :progress="progress"
-        class="completion-icon"
-      />
-      {{ coreString('completedLabel') }}
+      <template v-if="isQuiz">
+        <template v-if="!completed">
+          <ProgressIcon
+            :progress="progress"
+            class="completion-icon"
+          />
+          {{ inProgressLabel }}
+        </template>
+        <template v-else>
+          <ProgressIcon
+            :progress="progress"
+            class="completion-icon"
+          />
+          {{ completedLabel }}
+        </template>
+      </template>
+      <template v-else-if="completed">
```

Comment:
With the above tweak to the `v-if` this can just be a `v-else` instead.

Reply:
I added a ternary operator to display score or inProgress in a quiz, this eliminated the use of one more `else-if` statement.